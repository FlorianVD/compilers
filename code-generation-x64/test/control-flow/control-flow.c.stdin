1
2
3
4
5
9
37
456
-1
