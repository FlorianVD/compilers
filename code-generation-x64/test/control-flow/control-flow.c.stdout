> 1
> 1
> 2
> 3
> 5
> 34
> 24157817
> 1070169069571928416
> 